src/UI/CO_InfoController.java

```diff
@@ -106,7 +106,7 @@ public GameInstance getGame()
   }
 
   @Override
-  public ArrayList<CommanderInfo> getSelectedCOInfo()
+  public ArrayList<CommanderInfo> getSelectedCOInfoList()
```

Comment:
Why did this end up being actually necessary? And is the whole list selected (as the function name implies)?

Reply:
The `InfoController` can be used to check out CO info during game setup, which means you can't always fetch CO info by proxy of the current CO (since before the game, there's no CO).

I imagine the list-ness of it could start mattering if/when we start building Tag UI, but I don't think it matters at the moment.